src/components/shared/Button.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 function Button(props) {
   const { value, onClick } = props;
   return (
-    <button type="submit" value={value} onClick={onClick}>
+    <button className="btn" type="submit" value={value} onClick={onClick}>
       {value}
     </button>
   );
```

Comment:
- [ ] Kindly make this an arrow function to make sure your app is implemented in ES6 & to follow best practices.